File: springboot/open-api/src/main/java/com/openapi/domain/dto/resonse/UserAuthResponse.java
package com.openapi.domain.dto.resonse;

import lombok.Data;

/**
 * 登录/注册后返回的用户认证信息
 */
@Data
public class UserAuthResponse {
    private Long userId;
    private String account;
    private String name;
    private String avatarUrl;
    private String accessToken;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/UserAuthResponse.kt
package com.vectordemo.domain.dto.http.response

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

/** [userId] is decimal string on the wire (JSON); parse to Long for Room/session. */
data class UserAuthResponse(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    var userId: String? = null,
    var account: String? = null,
    var name: String? = null,
    var avatarUrl: String? = null,
    var accessToken: String? = null
)
